appveyor.yml

```diff
@@ -3,9 +3,15 @@
 # license that can be found in the LICENSE file.
 
 install:
-  - ps: .\tool\appveyor.ps1 install
+  - cinst -y dart-sdk
+  - ps: $env:PATH = "$env:PATH;c:\tools\dart-sdk\bin"
+  - ps: $env:PATH = "$env:PATH;c:\Users\appveyor\AppData\Roaming\Pub\Cache\bin"
```

Comment:
I'll update this (tha path). Not sure about the APPDATA usage; perhaps it's necessary for other reasons? Or, perhaps there was some unfamiliarity with windows conventions.